### PR TITLE
Notify when contacts join Signal

### PR DESCRIPTION
**Pull request by moxie0**
// FREEBIE

### CODE REVIEW

**Comment by heyimgay** [comment]
any chance we can get a screenshot? I was thinking we could add a notification button to the FAB (floating action button) with the new contacts count and maybe include them at the top (not on first run)

File: res/xml/syncadapter.xml [hunk 1]
```diff
@@ -3,6 +3,6 @@
               android:contentAuthority="com.android.contacts"
               android:accountType="org.thoughtcrime.securesms"
               android:userVisible="true"
-              android:supportsUploading="false"
+              android:supportsUploading="true"
```
Comment (mcginty) [inline res/xml/syncadapter.xml]: ?

Comment (moxie0) [inline res/xml/syncadapter.xml]: the system wasn't calling the sync adapter on a local modification within a reasonable amount of time otherwise

**Comment by moxie0** [comment]
in 3.2.0

**Pull request closed without merging**